numpy/core/tests/test_umath.py

```diff
@@ -185,6 +185,82 @@ def __array_wrap__(self, arr, context):
 
 
 class TestComparisons:
+    @pytest.mark.parametrize('dtype', np.sctypes['uint'] + np.sctypes['int'] +
+                             np.sctypes['float'] + [np.bool_])
+    def test_comparison_functions(self, dtype):
```

Comment:
```suggestion
    @pytest.mark.parametrize('dtype', np.sctypes['uint'] + np.sctypes['int'] +
                             np.sctypes['float'] + [np.bool_])
    @pytest.mark.parametrize('py_comp,np_comp', [
        (operator.lt, np.less),
        (operator.le, np.less_equal)
        (operator.gt, np.greater)
        (operator.ge, np.greater_equal)
        (operator.eq, np.equal)
        (operator.ne, np.not_equal)
    ])
    def test_comparison_functions(self, dtype, py_comp, np_comp):
```

Reply:
Done! Thank you! I tried this mapping but I didn't know I could use `operator.*`.